src/QMCDrivers/QMCDriverNew.h

```diff
@@ -304,7 +331,10 @@ class QMCDriverNew : public QMCDriverInterface, public MPIObjectBase
 
   /** }@ */
 
-        std::vector<RandomGenerator_t*> RngCompatibility;
+  std::vector<RandomGenerator_t*> RngCompatibility;
+
+  Timers timers_;
```

Comment:
Is it under `public:`? I think it should be in protected.

Reply:
Done